public/controllers/management/components/management/cdblists/components/columns.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { EuiToolTip, EuiButtonIcon, EuiBadge } from '@elastic/eui';
+import { resourceDictionary, ResourcesHandler, ResourcesConstants } from '../../common/resources-handler';
+import exportCsv from '../../../../../../react-services/wz-csv';
+import { WzButtonPermissions } from '../../../../../../components/common/permissions/button';
+import { WzButtonPermissionsModalConfirm } from '../../../../../../components/common/buttons';
+
+export default class CDBListsColumns {
+  constructor(props) {
+    this.props = props;
+
+    this.buildColumns = () => {
+      this.columns = {
+        lists: [
+          {
+            field: 'filename',
+            name: 'Name',
+            align: 'left',
+            sortable: true
+          },
+          {
+            field: 'relative_dirname',
+            name: 'Path',
+            align: 'left',
+            sortable: true
+          },
+          {
+            name: 'Actions',
+            align: 'left',
+            render: (item) => (
+              <EuiToolTip position="top" content={`Export ${item.filename}`}>
+                <EuiButtonIcon
+                  aria-label="Export list"
+                  iconType="exportAction"
+                  onClick={async ev => {
+                    ev.stopPropagation();
+                    await exportCsv(`/lists?path=${item.relative_dirname}/${item.filename}`,
+                      [{ _isCDBList: true, name: 'path', value: `${item.relative_dirname}/${item.filename}` }],
+                      item.filename
+                    )
```

Comment:
**nitpick**: We should take into account the action of the action buttons should manage the errors. Consider this: https://github.com/wazuh/wazuh-kibana-app/pull/4174#issuecomment-1175091824. This is suitable for Rules/Decoders/CDB Lists action buttons.